maintainers/maintainer-list.nix

```diff
@@ -14720,4 +14720,13 @@
     github = "dfithian";
     githubId = 8409320;
   };
+  MatthieuBarthel = {
+    email = "matthieu@imatt.ch";
+    name = "Matthieu Barthel";
+    github = "MatthieuBarthel";
+    githubId = 435534;
+    keys = [{
+      fingerprint = "80EB 0F2B 484A BB80 7BEF  4145 BA23 F10E AADC 2E26";
+    }];
+  };
```

Comment:
The entries should be sorted. Please move it in an appropriate place.

Reply:
oups :) thanks for the review